content/concepts/consensus-network/amendments/contribute-code-flow.md

```diff
@@ -0,0 +1,82 @@
+---
+html: contribute-code-flow.html
+parent: amendments.html
+blurb: 
+labels:
+  - Blockchain
+---
+# Contribute Code to the XRP Ledger
+
+The software that powers the XRP Ledger is open-source, so anyone can download, modify, extend, or explore it. If you want to contribute code, it's important to work with the community in defining the specifications of your changes and testing the code before adding it to `rippled`.
+
+
+## Standards Draft
+
+Most changes will start with a standards draft. The standards draft outlines the specifications of your change and provides the XRPL community a chance to give feedback. Before committing to development, always submit a standards draft to the [XRPL-Standards repo](https://github.com/XRPLF/XRPL-Standards/discussions/categories/standard-proposals).
+
+**Note:** Bug fixes don't require standards drafts, but may require amendments.
+
+Your standards draft should supply the community with information to facilitate useful discussion. Helpful sections to include are:
+
+- Abstract/Introduction
+- Specifications
+- Considerations
+
+***TODO: Link to a standards draft outline.***
```

Comment:
Not sure if there's a template for the outline, but all standards start as discussions on https://github.com/XRPLF/XRPL-Standards

Reply:
There is no template, but we could create one. Maybe add a line that you can reference the XRPL-Standards README for the draft requirements and lifecycle?

Reply:
A template would be useful to include in the README to standardize drafts.